presto-orc/src/main/java/com/facebook/presto/orc/writer/ColumnWriter.java

```diff
@@ -58,6 +58,11 @@
 
     long getBufferedBytes();
 
+    /**
+     * Estimated output data size takes compression into account.
+     */
+    long estimateOutputDataSize();
```

Comment:
Instead of add ing a new method, why not rename reuse/rename the existing `getBufferedBytes` method?  I believe this is the entire point of the `getBufferedBytes` method